Dockerfile

```diff
@@ -30,36 +39,27 @@ RUN apt-get update && \
     add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu $(lsb_release -cs)-cran40/" -y && \
     apt-get update && \
     apt-get install --no-install-recommends -y \
-        git \
-        gcc \
+        dart \
+        dotnet-sdk-5.0 \
         g++ \
-        libc-dev \
-        libgomp1 \
-        python${python}-dev \
-        python3-setuptools \
-        python3-pip \
-        zulu-8 \
+        gcc \
+        git \
         golang-go \
-        dotnet-sdk-5.0 \
+        haskell-platform \
+        php \
         powershell \
+        python${python}-dev \
+        python3-pip \
+        python3-setuptools \
         r-base \
-        php \
-        dart \
-        haskell-platform \
-        ruby-full && \
+        ruby-full \
+        zulu-8 && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /m2cgen
 
 COPY requirements-test.txt ./
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python${python} 1 && \
     python -m pip install --upgrade pip && \
-    pip install --no-cache-dir Cython numpy && \
+    pip install --no-cache-dir Cython "numpy==1.19.2" && \
```

Comment:
`lightning` package need to be compiled with the same `numpy` version as one used in tests to prevent the following error:
```
__init__.pxd:242: in init lightning.impl.randomkit.random_fast
    ???
E   ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```